### PR TITLE
Update feature flag settings

### PR DESCRIPTION
**Pull request by sheelio**
Makes the following feature flags project only (no per user setting):
- Case List Tile
- Case List Lookup
- Multimedia Case Properties
- UCR

Change categories for:
- Mobile Privileges (now Internal)
- Mobile Worker Self Registration (now Custom)
- SMS Performance Feedback (now Custom)
- Anonymous Web Apps Usage (now Custom)

### CODE REVIEW

**Comment by sheelio** [comment]
@kaapstorm 
Can you take a look at this?

**Review by kaapstorm — APPROVED** [review]

**Comment by kaapstorm** [comment]
LGTM

File: corehq/toggles.py [hunk 1]
```diff
@@ -477,7 +477,7 @@ def toggle_values_by_name(username=None, domain=None):
     'user_reports',
     'User configurable reports UI',
     TAG_SOLUTIONS,
-    [NAMESPACE_DOMAIN, NAMESPACE_USER],
+    [NAMESPACE_DOMAIN],
```
Comment (emord) [inline corehq/toggles.py]: @sheelio Why did you change this? I think it's going to make it much harder for us to debug UCRs if we can't go look at them

Comment (sheelio) [reply corehq/toggles.py]: If the project is in the feature flag, you would still have access right?

Comment (emord) [reply corehq/toggles.py]: I'm generally pro domain namespace only, but I think this can also be helpful to debug report builder reports, and being able to rebuild tables manually on any domain.

If we give the access to the domain, then the "edit report" button would show up for all users in that domain (likely bad for enikshay/icds right?)
